nuitka/build/inline_copy/lib/scons-2.3.2/SCons/Platform/win32.py

```diff
@@ -130,8 +130,10 @@ def piped_spawn(sh, escape, cmd, args, env, stdout, stderr):
         return 127
     else:
         # one temporary file for stdout and stderr
-        tmpFileStdout = os.path.normpath(tempfile.mktemp())
-        tmpFileStderr = os.path.normpath(tempfile.mktemp())
+        tmpFileStdout, tmpFileStdoutName = tempfile.mkstemp(text=True)
```

Comment:
I don't see how this is any safer, after closing, the race is back on. Note that this code is used with Python 2.6 only.

Reply:
The `tempfile.mkstemp` method is available and works for Python 2 too.

Reply:
Yes, that's true, I had checked myself too, also to see if `text` is a good mode, of that I wasn't sure, but since it's only using the filename, the mode doesn't really matter anyway. So not sure why it's given.